packages/host/app/components/operator-mode/stack.gts

```diff
@@ -37,6 +37,17 @@ export default class OperatorModeStack extends Component<Signature> {
     for (let i = this.args.stackItems.length - 1; i > itemIndex; i--) {
       itemsToDismiss.push(this.args.stackItems[i]);
     }
+
+    // add closing animation on last item
+    const lastItemIndex = this.args.stackItems.length - 1;
+    const lastItemEl = document.querySelector(
+      `[data-stack-card-index="${lastItemIndex}"]`,
+    ) as HTMLElement;
+    if (lastItemEl) {
+      lastItemEl.classList.add('closing-animation');
```

Comment:
this is super awkward how we reach into another component and set a class on it. let's come of with a less promiscuous solution here. the stack item itself needs to be the one that sets classes on itself. There already is a `setupStackItem()` function in the args that we can use to have clear API between the `Stack` and it's `StackItems`. Take a look at how `clearSelections()`, `doScrollIntoView()`, and `doWithStableScroll()` work. All these methods are API that the `StackItem` shares with the `Stack`, such that when a `Stack` needs one of it's `StackItems` to do these things, it can just call this method on the `StackItem` instead of reaching into the internals of the `SackItem`, and thus leaking the StackItem's abstractions